integration_tests/test_employment.py

```diff
@@ -73,3 +75,80 @@ def test_movers_change_employment(populations):
         assert (
             before[should_change]["employer_id"] != after[should_change]["employer_id"]
         ).all()
+
+
+def test_employment_income_propensity_updates(simulants_on_adjacent_timesteps):
```

Comment:
This test and the next can be combined pretty easily: make a boolean for whether employment changed, and a boolean for whether the employer_income_propensity changed, and assert that those two boolean Series are always equal.